draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,222 @@
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1313651865648369725}{Archon Studio Discord}
+
+\textit{``Sir! The raiders are at our gates!''}
+
+\textit{``Sound the warhorn! We'll drive them back as always!'' you command, and go to defend the city. Just as you have done many times before. Such is the harsh life of the border clans, where wars and skirmishes with neighboring rivals never truly end. Fortunately, your alliance remains strong, and you know your back is covered.}
+
+\textit{The time has come to end these long-standing wars and crush your enemies like a bug once and for all!}
+% Translator info: In Homam IV was Quote like: "Rufux will crush this army like a bug." If has scounting and has bigger army. So "like a bug" is little easter egg referencing on this.
+
+\subsection*{\MakeUppercase{Scenario Length}}
+
+This Scenario is played over 7 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+
+\textbf{Player Count:} 1--6
+
+\textbf{Starting Resources:} 14 \svg{gold}, 5 \svg{building_materials}, 2 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item  1 × A Few \svgunit{bronze} Units of your choice
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling.
+
+\textbf{Map Tile Pool:} Take $P$ × random Far (II–-III) Map Tiles to form a shared pool, where $P$ equals the number of players. None of these Tiles may contain a Settlement.
+
+\subsection*{\MakeUppercase{Map Setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout.
+Place Far (II--III) Map Tiles with Settlements that match your Faction's Towns. Mark these with a colored Cube (see example).
+
+\begin{itemize}
+  \item $P$ × Starting (I) Map Tile.
+  \item $P$ × Far (II--III) Map Tile with Settlement of same Faction as players Towns.
+  \item 2$P$ × Far (II--III) Map Tile - must not contain Tiles with Settlements.
+\end{itemize}
+
+\textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}
+
+\subsection*{\MakeUppercase{AI Hero Setup}}
+
+Prepare an AI Deck for Settlement defenders.
+
+\textbf{Settlement defenders Deck:} 1 × Attack Card, 1 × Defense Card, 1 × Spell Card
+
+\textbf{Settlement defenders Spell Deck:} 1 × Magic Arrow
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+Conquer all Settlements.
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+
+\begin{itemize}
+  \item Any player loses their Town.
+  \item At least one Settlement remains unclaimed by the end of the \nth{7} Round. Time has run out!
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed Events}}
+    \textbf{\nth{1} Round:}
+    \begin{itemize}
+      \item At start of the Round, each player show up the \textit{nemesis map Tiles} (Far (II--III) Map Tile with Settlement of same Faction as players Towns). This Tile you can reveal freely.
```

Comment:
```suggestion
  \textbf{\nth{1} Round:}
  \begin{itemize}
    \item At the start of each Round, all players reveal their \textit{Nemesis Map Tiles} (Far Map Tiles (II--III) containing Settlements that match their Town's home terrain). These Tiles may be revealed without any cost.
```

Reply:
I has to disagree with this nnew wording. 


At the start of  **THE** Round, all players reveal their \textit{Nemesis Map Tiles} (Far Map Tiles (II--III) containing Settlements that match their Town's home terrain). These Tiles may be revealed without any cost.

not eahc round, only first round.. maybe you wanted say "**each Turn**"?

Reply:
Yes, sorry, fixed.